### PR TITLE
Create README.md platform

### LINKED ISSUE #4534: add helm commands for all hmda deployments
- [ ] hmda-platform
- [ ] hmda-analytics
- [ ] hmda-dashboard
- [ ] hmda-data-publisher
- [ ] hmda-quarterly-data-service
- [ ] hmda-reporting
- [ ] check-digit
- [ ] data-browser
- [ ] email-service
- [ ] file-proxy
- [ ] institutions-api
- [ ] irs-publisher
- [ ] modified-lar
- [ ] ratespread-calculator

### PR DESCRIPTION
**Pull request by zencircle**
closes #4534

### CODE REVIEW

File: kubernetes/hmda-platform/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+kubectl create secret generic inst-postgres-credentials --from-literal=username= --from-literal=password= --from-literal=host= --from-literal=url="jdbc:postgresql://postgresql:5432/hmda?user= &password= &sslmode=false"
+```
+- Configmap
+```
+kubectl apply -f https://github.com/cfpb/hmda-platform/tree/master/kubernetes/config-maps
+```
+
+Update
+```
+helm upgrade --install --force --namespace=default --values=kubernetes/hmda-platform/values-dev.yaml \
```
Comment (lchen-2101) [inline kubernetes/hmda-platform/README.md:18]: I'd probably remove `--force` and see the issues before deciding to "force" it; if there are issues.

File: kubernetes/hmda-platform/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+--set image.tag=latest \
+--set service.name=hmda-platform \
+hmda-platform kubernetes/hmda-platform
+```
+
+Delete
+```
+kubectl delete secret cassandra-credentials
+kubectl delete secret generic inst-postgres-credentials
```
Comment (lchen-2101) [inline kubernetes/hmda-platform/README.md:27]: deletion don't have `generic` keyword

File: kubernetes/hmda-platform/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+kubectl delete -f https://github.com/cfpb/hmda-platform/tree/master/kubernetes/config-maps
+helm uninstall hmda-platform
```
Comment (lchen-2101) [inline kubernetes/hmda-platform/README.md:29]: probably uninstall first before deleting the secrets and cms

**Comment by PatrickGoRaft** [comment]
@lchen-2101  @zencircle  have all the concerns for this PR been resolved?

File: kubernetes/hmda-platform/README.md (hunk 2 above)
Comment (lchen-2101) [reply kubernetes/hmda-platform/README.md:28]: not sure if it's github, but I'm still seeing `generic` in this secret deletion.

**Pull request merged**